tool/canonical/fix_excludes.json

```diff
@@ -0,0 +1,94 @@
+[
+  {
+    "lint" : "avoid_renaming_method_parameters",
+    "notes" : "naming"
+  },
+  {
+    "lint" : "avoid_shadowing_type_parameters",
+    "notes" : "naming"
+  },
+  {
+    "lint" : "camel_case_extensions",
+    "notes" : "naming"
+  },
+  {
+    "lint" : "camel_case_types",
+    "notes" : "naming"
+  },
+  {
+    "lint" : "constant_identifier_names",
+    "notes" : "naming"
+  },
+  {
+    "lint" : "control_flow_in_finally",
+    "notes" : "TODO"
+  },
+  {
+    "lint" : "exhaustive_cases",
+    "notes" : "infeasible"
```

Comment:
On the other hand I think this one is very fixable. We know the valid values and can add the missing case labels with `break`s. Maybe this was pasted into the wrong place?

Reply:
whoops.  yep.  good catch!